.github/workflows/release-test.yml

```diff
@@ -3,154 +3,80 @@ name: Release test Workflow
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
         with:
-          ref: ${{ env.GITHUB_REF }}
-          fetch-depth: 0
-          submodules: recursive
-      - uses: ./.github/actions/init
-      - name: Visual Regression Tests
+          path: ~/.npm
+          key: npm-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            npm-node-
+      - name: Auth Git
+        run: |
+          git config user.email "${{ env.GIT_EMAIL }}"
+          git config user.name "${{ env.GIT_USER }}"
+      - name: Install Deps
+        run: npm ci
+      - name: Build packages
+        run: npx nx run-many --target=build --all --parallel=2 --exclude testfront, storybook, fulfillment, storefront
+      - name: Fix js extension
+        run: node tools/post-build/add-js-extension.js
+      - name: Install verdaccio
+        run: npm i -g verdaccio
+      - name: Run verdaccio first time to generate config
+        run: verdaccio &
+      - name: Wait for verdaccio to start up
+        run: |
+          while ! nc -z localhost 4873; do
+            sleep 1
+          done
+      - name: Stop verdaccio
+        run: kill -9 $(lsof -ti :4873)
+      - name: Allow to publish packages to everyone for verdaccio
+        run: |
+          if [ ! -f "$VERDACCIO_CONFIG" ]; then
+            echo "Creating Verdaccio config file: $VERDACCIO_CONFIG"
+            mkdir -p $(dirname "$VERDACCIO_CONFIG")
+            touch "$VERDACCIO_CONFIG"
+          fi
+          sed -i 's/\$authenticated/\$all/g' $VERDACCIO_CONFIG
         env:
-          CHROMATIC_PROJECT_TOKEN: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          CHROMATIC_SHA: ${{ github.event.after }}
-          CHROMATIC_BRANCH: development
-        run: npx nx run storybook:visual-regression-branch
-  test-e2e-storefront:
-    if: ${{ github.actor != 'dependabot[bot]' }}
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          ref: ${{ env.GITHUB_REF }}
-          fetch-depth: 0
-          submodules: recursive
-      - uses: ./.github/actions/init
-      - name: Storefront E2E Tests
-        uses: ./.github/actions/run-optimal
+          VERDACCIO_CONFIG: /home/runner/.config/verdaccio/config.yaml
+      - name: Run verdaccio gain
+        run: verdaccio &
+      - name: Set npm registry for @spryker-oryx to verdaccio
+        run: npm config set @spryker-oryx:registry http://localhost:4873
+      - name: Release minor
+        run: npm run release-minor
+      - name: Install dependencies for testfront
+        run: cd apps/testfront && npm i && npm list --depth=0 && cd ../..
+      - name: Run testfront
+        run: npm run testfront
         env:
-          SCOS_BASE_URL: ${{ secrets.SCOS_BASE_URL }}
-          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          FES_CONTENT_BACKEND_URL: https://test-experience-builder.netlify.app/.netlify/functions/server
```

Comment:
hardcoded URLs - should we remove them to ENV vars?

Reply:
Thanks, fixed.